src/sqltest/parts/query/editor/queryEditor.test.ts

```diff
@@ -343,25 +303,15 @@ suite('SQL QueryEditor Tests', () => {
 			queryModelService.setup(x => x.isRunningQuery(TypeMoq.It.isAny())).returns(() => false);
 			// If I use the created QueryEditor with no changes since creation
 			// Buttons should be set as if disconnected
-			assert.equal(queryInput.runQueryEnabled, true, 'runQueryAction button should be enabled');
-			assert.equal(queryInput.cancelQueryEnabled, false, 'cancelQueryAction button should not be enabled');
-			assert.equal(queryInput.connectEnabled, true, 'connectDatabaseAction button should be enabled');
-			assert.equal(queryInput.disconnectEnabled, false, 'disconnectDatabaseAction button should not be enabled');
-			assert.equal(queryInput.changeConnectionEnabled, false, 'changeConnectionAction button should not be enabled');
-			assert.equal(queryInput.listDatabasesConnected, false);
+			assert.equal(queryInput.state.connected, false, 'runQueryAction button should be enabled');
 			done();
 		});
 
 		test('Taskbar buttons are set correctly upon connect', (done) => {
 			let params: INewConnectionParams = { connectionType: ConnectionType.editor, runQueryOnCompletion: RunQueryOnConnectionMode.none };
 			queryInput.onConnectSuccess(params);
 			queryModelService.setup(x => x.isRunningQuery(TypeMoq.It.isAny())).returns(() => false);
-			assert.equal(queryInput.runQueryEnabled, true, 'runQueryAction button should be enabled');
-			assert.equal(queryInput.cancelQueryEnabled, false, 'cancelQueryAction button should not be enabled');
-			assert.equal(queryInput.connectEnabled, false, 'connectDatabaseAction button should not be enabled');
-			assert.equal(queryInput.disconnectEnabled, true, 'disconnectDatabaseAction button should be enabled');
-			assert.equal(queryInput.changeConnectionEnabled, true, 'changeConnectionAction button should be enabled');
-			assert.equal(queryInput.listDatabasesConnected, true);
+			assert.equal(queryInput.state.connected, true, 'runQueryAction button should be enabled');
```

Comment:
>runQueryAction button should be enabled [](start = 51, length = 39)

The message is no longer accurate. Seems like you should also be checking the other properties on state (connecting, executing, resultsVisible, etc)